sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -562,6 +562,12 @@ def _add_argparse_args(cls, parser):
         default=None,
         choices=['COST_OPTIMIZED', 'SPEED_OPTIMIZED'],
         help='Set the Flexible Resource Scheduling mode')
+    parser.add_argument(
```

Comment:
Looking at the Java implementation (https://github.com/apache/beam/pull/7047), this is an experiment and not a top level option.

I believe you can handle this similar to other experiments in internal/apiclient.py

Reply:
Re option, ok, I will handle it as experiment.